mobile/src/employee-details/employee-details-events-list.tsx

```diff
@@ -26,6 +26,11 @@ export class EmployeeDetailsEventsList extends Component<EmployeeDetailsEventsLi
     private readonly eventDigitsDateFormat = 'DD/MM/YYYY';
 
     public render() {
+
+        const events = this.props.events 
+            ? this.props.events.sort((a, b) => a.dates.startDate.valueOf() - b.dates.startDate.valueOf()) 
```

Comment:
maybe do it on the server-side?

Reply:
of course not, it's a view-specific feature. Even if server returned it sorted, we'd better sort it anyway